code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/impl/ExternalProjectResponse.java

```diff
@@ -0,0 +1,16 @@
+package io.cattle.platform.servicediscovery.service.impl;
```

Comment:
it's better to not put it to the separate file; either put it to regionutil, or regionserviceimpl